### DISCUSSION TITLE
Is there any limitation MQTT Bridge with AWS ?

### DISCUSSION THREAD

**Discussion by kunalfleethawks** [emqx/emqx#8020] [Q&A] [answered]
Hello Team,

If we bridge EMQX wtih AWS IOT Core to republish all messages, Is there any limits if we send millions of message to EMQX?

**Comment by zmstone** [comment, marked as answer]
Hi @kunalfleethawks 

No there is no limit in number of messages.

The enterprise edition has number of client connection limit.
The community edition has no such limit.
Concurrent connection/session limit is the only forced limit on EMQX Enterprise because our pricing model base on it.

Other than that, there are other limits mostly for system stability protection.
* Connection rate limit
* Messaging rate limit
* Packet size limit
Quite a few more, but mostly are configurable though.

**Reply by kunalfleethawks** [reply]
Thank you for the clarification. Is there any limitation from the AWS IOT Core side as well?

**Reply by zmstone** [reply]
Sorry I’m not sure how aws iot limits messages
